src/main/java/org/bih/eos/converter/cdt/converter/configurable/specific/ObservationConverter.java

```diff
@@ -115,10 +128,18 @@ private ObservationEntity convertDataValues(ObservationEntity observation, Eleme
             if (element.getValue().getClass().equals(DvQuantity.class)) {
                 observation.setValue(DvGetter.getDvQuantity((DvQuantity) element.getValue()));
             } else if (element.getValue().getClass().equals(DvText.class)) {
-                observation.setValue(DvGetter.getDvText((DvText) element.getValue()));
+            	Optional<String> dvText = DvGetter.getDvText((DvText) element.getValue());
```

Comment:
method now too big, please split into smaller ones that are more readable. 
try to reach cogntivie complexity of 3-5
https://docs.codeclimate.com/docs/cognitive-complexity